File: Contests/CPC UNJFSC/FIISI Code Sep 2024/A. Satélites en Órbita.java
import java.util.*;

public class Main {
  public static void main(String[] args) {
    Scanner scanner = new Scanner(System.in);
    // leer numero de casos
    int t = scanner.nextInt();
    scanner.nextLine();
    
    for (int i = 0; i < t; i++) {
      solve(scanner);
    }
    
    scanner.close();
  }

  private static void solve(Scanner scanner) {
    // leer la cadena de caracteres
    String cadena = scanner.nextLine();
    int counter = 0;

    // contar las ocurrencias de 'O'
    for (int i = 0; i < cadena.length(); i++) {
      if (cadena.charAt(i) == 'O') {
        counter++;
      }
    }

    System.out.println(counter);
  }
}

File: Contests/CPC UNJFSC/FIISI Code Sep 2024/A. Satélites en Órbita.cpp
#include <bits/stdc++.h>

using namespace std;

void solve() {
  string cadena; cin >> cadena;
  int counter = 0;
  for (int i=0; i<cadena.size(); i++) {
    if (cadena[i] == 'O') {
      counter++;
    }
  }
  cout << counter << endl;
}

int main() {
  int numCasos; cin >> numCasos;
  while (numCasos--) {
    solve();
  }

  return 0;
}